### DISCUSSION TITLE
Boot loop

### DISCUSSION THREAD

**Discussion by ParkerGit2328** [helloSystem/hello#565] [General]
I installed Hellosystem on my HP Stream and when I booted it, it ran into a boot loop. How do I fix that?

**Comment by probonopd** [comment]
Please boot in verbose mode and try to catch/film the last messages before the reboot. https://hellosystem.github.io/docs/developer/boot.html#boot-in-verbose-mode

Does your computer have an SD card reader? Try to boot with a card inserted (there seems to be a bug causing reboots when there is an empty card reader detected).
